components/component-stylesheets/cards.css

```diff
@@ -0,0 +1,145 @@
+.card{
+    display: flex;
+    flex-direction: column;
+    max-width: 26rem;
+    border-radius: 10px;
+    overflow: hidden;
+    position: relative;
+    box-shadow: 0 3px 10px var(--mr-light-background-color);
+}
+.main{
+    margin-top: 20px;
+}
+
+.card-info{
+    margin: 0 1.5rem 1.8rem;
+}
+
+.card-header{
+    font-size: 1.5rem;
+    font-weight: 500;
+}
+
+.card-author{
+    font-size: 1.2rem;
+    font-weight: 500;
+    color: var(--mr-dark-default);
+    font-weight: 400;
+}
+
+.card-badge{
+    position: absolute;
+    top: 10px;
+    background-color: var(--mr-primary-color);
+    color: var(--mr-white-text-color);
+    padding: 0.2rem 0.8rem;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.card-about{
+    font-size: 1.2rem;
+    font-weight: 400;
+    color: var(--mr-dark-default);
+    margin-top: 1rem;
+    font-weight: 700;
```

Comment:
you are overwriting font-weight here.

Reply:
thanks for pointing out, fixed in recent commit